docs/src/documentation/05-development/04-migration-guides/01-v16.mdx

```diff
@@ -10,16 +10,42 @@ redirect_from:
 This migration guide focuses on the process of migrating from Orbit v15 to v16.0, as some breaking changes were introduced.
 With this guide, we aim to walk through all the breaking changes and how they can be addressed, allowing the migration to be smoother and effortlessly.
 
-## Breaking changes
-
-### Token update (WIP -- Update this section as needed to document each change)
+## Token update
 
 A number of new design tokens were introduced.
 
 Here's the mapping:
 
 - `rounded-small` -> `rounded-50`
-- ...
+- `rounded-normal` -> `rounded-100`
```

Comment:
Maybe we can add a note here saying that the value is expectedly different. Something like

```suggestion
- `rounded-normal` -> `rounded-100` (value change from 3px to 4px)
```

Reply:
yes, sounds cool. also, we can add similar note to rounded-circle (value changed from 50% to 9999px). WDYT?

Reply:
No need. That is unperceivable by the users